projects/packages/forms/src/contact-form/class-admin.php

```diff
@@ -888,9 +888,9 @@ public function grunion_manage_post_row_actions( $actions ) {
 			);
 			$actions['trash'] = sprintf(
 				'<a class="submitdelete" title="%s" href="%s">%s</a>',
-				esc_attr__( 'Trash', 'jetpack-forms' ),
+				esc_attr_x( 'Trash', 'verb', 'jetpack-forms' ),
```

Comment:
I wonder if this one, used for the title attribute, could be changed to "Move to Trash" or something along those lines.